### DISCUSSION TITLE
Apakah AWS memenuhi syarat PJP BI?

### DISCUSSION THREAD

**Discussion by mu-hasan** [awsugid/forum#39] [General]
Halo semua. Perkenalkan, saya Hasan. Saya diminta setup cloud service untuk kebutuhan salah satu perusahaan Payment Gateway. Dari pihak perusahaan, yg mereka tau hanya GCP dan Alibaba Cloud yg memenuhi persyaratan dari BI untuk bisa dapat PJP. Mungkin rekan-rekan disini ada yang tau atau punya pengalaman, apakah AWS juga bisa memenuhi persyaratan untuk mengurus PJP ke BI? Terimakasih 🙌

**Comment by arifsetyawan** [comment]
AWS memenuhi syarat untuk dipakai oleh PJP. berdasarkan informasi di  https://aws.amazon.com/financial-services/security-compliance/compliance-center/id/
